www/views/confirm.html

```diff
@@ -7,7 +7,7 @@
     </ion-nav-back-button>
   </ion-nav-bar>
 
-  <ion-content ng-class="{'slide-to-pay': !hasClick && !insuffientFunds && !noMatchingWallet}">
```

Comment:
I think we want to be determining this functionality based on device capabilities, rather than the specific platform. Maybe @msalcala11 should look at this too?